### DISCUSSION TITLE
Higher CPU usage after 0.14?

### DISCUSSION THREAD

**Discussion by Marcos-kp** [blakeblackshear/frigate#12929] [General]
Hey! After upgrading to the latest version, my CPU usage has been up by quite a bit. I always run my config on the highest I can, cpu hitting 80%to95% of usage sometimes.
After the update (and checking masks) my cpu usage is hitting 100% and caused frigate to lose camera frames and drop all images. Anyone having the same issue?

![IMG_1630](https://github.com/user-attachments/assets/2c5c4d86-441a-4ab6-9f27-c4eab2038225)
![IMG_1631](https://github.com/user-attachments/assets/b503b174-ec3f-4c07-b48f-49282fe1baff)

Same time different day, same cameras with detectors on, same weather conditions.

**Comment by NickM-27** [comment]
Most feedback we've gotten has said it's the same or lower. We would need to see screenshots of the system and cameras metrics pages

**Reply by f1d094** [reply]
As another data point: I also was seeing a significant increase in CPU after upgrade from 0.13.2 to 0.14.0, around  ~30%. [It was more than my system could handle](https://github.com/blakeblackshear/frigate/discussions/13002#discussioncomment-10328691). I wound up rolling back to 0.13.2 as a result and am back down to 60-70% util.
